Gems/AWSCore/Code/Tools/ResourceMappingTool/resource_mapping_tool.py

```diff
@@ -40,6 +43,23 @@
 logger = logging.getLogger(__name__)
 logger.info(f"Using {logging_path} for logging.")
 
+# Optional executable path needed on platforms that need to preload Pyside2 (or other) related
+# shared libraries in order for the pyside python library to load the correct one
+if platform.system() == 'Linux':
+
+    import ctypes
+
+    if not arguments.executable_path:
+        logger.error(f"Argument '--executable-path' is required on platform: {platform.system()}")
+        exit(-1)
+    preload_shared_libs = [f'{arguments.executable_path}/libpyside2.abi3.so.5.14',
+                           f'{arguments.executable_path}/libQt5Widgets.so.5']
+    for preload_shared_lib in preload_shared_libs:
+        if not os.path.exists(preload_shared_lib):
+            logger.error(f"Cannot find required shared library at {preload_shared_lib}")
+            exit(-1)
+        ctypes.CDLL(preload_shared_lib)
+
```

Comment:
I think you can combine this under environment util, and please add corresponding unit test

Reply:
Already refactored it to that file. Where are the unit tests?

Reply:
unit tests are under tests folder

Reply:
Updated tests